docs/topic_guides/comparators.md

```diff
@@ -3,25 +3,166 @@ tags:
 
-## Jaro similarity
+For guidance on how to choose the most suitable string comparator, and associated threshold, see the dedicated [topic guide]().
 
+<hr>
 
-The Jaro similarity measure is a string similarity measure that quantifies the similarity between two strings based on the number of common characters between them and their relative position. It is often used in applications such as entity resolution and duplicate detection, and is particularly useful for comparing strings that are similar in length and contain similar characters.
+## Levenshtein Distance
 
-The formula for Jaro similarity is as follows:
+**Useful for:** Data entry errors e.g. character miskeys.  
+**Splink comparison functions:** [levenshtein_level()](../comparison_level_library.md#splink.comparison_level_library.LevenshteinLevelBase) and [levenshtein_at_thresholds()](../comparison_library.md#splink.comparison_library.LevenshteinAtThresholdsComparisonBase)  
+**Returns:** An integer (lower is more similar).
+
+### Description
+Levenshtein distance, also known as edit distance, is a measure of the difference between two strings. It represents the minimum number of **insertions**, **deletions**, or **substitutions** of characters required to transform one string into the other.
+
+Or, as a formula,
+
+$$Levenstein(s_1, s_2) = \min \lbrace \begin{array}{l}
+\text{insertion} \ ,
+\text{deletion} ,
+\text{substitution} 
+\end{array} \rbrace $$
+
+### Examples
+
+**"KITTEN" vs "SITTING"**
+
+The minimum number of operations to convert "KITTEN" into "SITTING" are:
+
+- Substitute "K" in "KITTEN" with "S" to get "SITTEN."
+- Substitute "E" in "SITTEN" with "I" to get "SITTIN."
+- Insert "G" after "N" in "SITTIN" to get "SITTING."
+
+Therefore, 
+
+$$Levenstein(KITTEN, SITTING) = 3$$
+
+<br>
+
+**"CAKE" vs "ACKE"**
+
+The minimum number of operations to convert "CAKE" into "ACKE" are:
+
+- Substitute "C" in "CAKE" with "A" to get "AAKE."
+- substitute the second "A" in "AAKE" with "C" to get "ACKE."
+
+Therefore, 
+
+$$Levenstein(CAKE, ACKE) = 2$$
+
+### Sample code
+
+You can test out the Damerau-Levenshtein distance between two strings through the [jellyfish](https://jamesturk.github.io/jellyfish/) package.
+
+```python
+import jellyfish
+levenshtein_distance("CAKE", "ACKE)
+```
+> 2
+
+<hr>
+
+## Damerau-Levenshtein Distance
+
+**Useful for:** Data entry errors e.g. character transpositions and miskeys  
+**Splink comparison functions:** [damerau_levenshtein_level()](../comparison_level_library.md#splink.comparison_level_library.DamerauLevenshteinLevelBase) and [damerau_levenshtein_at_thresholds()](../comparison_library.md#splink.comparison_library.DamerauLevenshteinAtThresholdsComparisonBase)  
```

Comment:
Damerau-Levenshtein comparison and comparison level are not in the docs yet. Not sure if it's in a separate PR cos it's pretty new

Reply:
Yea Damearau-Levenshtein is sitting on [this PR](https://github.com/moj-analytical-services/splink/pull/1181) waiting for duckdb to release a new package version. I thought there would have been a release by now so this was a placeholder for when that comes in.
I am tempted to leave this as is and then it will become live when damerau levenshtein is merged

Reply:
I agree, just flagging in case it wasn't written up yet or wouldn't be released for a while.